src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Prop = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Prop> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li>
+        <MovieCard key={movie.imdbId} movie={movie} />
```

Comment:
`key` attr must always be at the top level of map result. In this case - in `li` tag.